secator/report.py

```
@@ -76,13 +76,30 @@ def build(self, extractors=[], dedupe=CONFIG.runners.remove_duplicates):
 			del data['info']['results']
 		data['info']['title'] = self.title
 		data['info']['errors'] = self.runner.errors
+		data['info']['celery_id'] = self.runner.celery_result.id if self.runner.celery_result else None
+		data['info']['celery_ids'] = self.runner.celery_ids
+		ids_map = self.runner.celery_ids_map.copy()
+		for idx, val in ids_map.items():
```

Ruff (0.15.9)

[warning] Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)